### PR TITLE
fix wrong description about hstore algo

### PR DESCRIPTION
**Pull request by Gustry**
@havatv Just follow up after your last commits about HStore.

### CODE REVIEW

File: source/docs/user_manual/processing_algs/qgis/vectortable.rst [hunk 1]
```diff
@@ -585,17 +587,11 @@ Parameters
...
-       
-       * Create Temporary Layer
-       * Save to File...
-       * Save to Geopackage...
-       * Save to PostGIS Table...
-       
-       The file encoding can also be changed here.
+
+       Default: empty
+     - List of fields separated by a comma to extract. The
```
Comment (havatv) [inline source/docs/user_manual/processing_algs/qgis/vectortable.rst]:
```suggestion
     - Comma-separated list of fields to extract.
```

File: source/docs/user_manual/processing_algs/qgis/vectortable.rst [hunk 2]
```diff
@@ -585,17 +587,11 @@ Parameters
...
+HStore field will be updated by removing these keys.
```
Comment (havatv) [inline source/docs/user_manual/processing_algs/qgis/vectortable.rst]:
```suggestion
       The HStore field will be updated by removing these keys.
```

File: source/docs/user_manual/processing_algs/qgis/vectortable.rst [hunk 3]
```diff
@@ -585,17 +587,11 @@ Parameters
...
-       One of:
-       
-       * Create Temporary Layer
-       * Save to File...
-       * Save to Geopackage...
-       * Save to PostGIS Table...
-       
-       The file encoding can also be changed here.
+
+       Default: empty
```
Comment (havatv) [inline source/docs/user_manual/processing_algs/qgis/vectortable.rst]:
```suggestion
       Default: ''
```

File: source/docs/user_manual/processing_algs/qgis/vectortable.rst [hunk 4]
```diff
@@ -558,7 +558,9 @@ and the HStore field is updated.
 By default, all unique keys are added.
 
 `HStore <https://www.postgresql.org/docs/10/hstore.html>`_
```
Comment (havatv) [inline source/docs/user_manual/processing_algs/qgis/vectortable.rst]:
```suggestion
The PostgreSQL `HStore <https://www.postgresql.org/docs/10/hstore.html>`_
```

File: source/docs/user_manual/processing_algs/qgis/vectortable.rst [hunk 5]
```diff
@@ -558,7 +558,9 @@ and the HStore field is updated.
...
-is a simple key-value store used in PostgreSQL.
+is a simple key-value store used in PostgreSQL and in OGR (when reading an
```
Comment (havatv) [inline source/docs/user_manual/processing_algs/qgis/vectortable.rst]:
```suggestion
is a simple key-value store used in PostgreSQL and OGR (when reading an
```

**Comment by Gustry** [comment]
Thanks for merging.

Sorry, I got nearly more suggestions from you that number of lines I edited in this PR.

BTW, I was a little bit surprised that we need to repeat in every algorithm the description of the output:
```
Specify the output vector layer. One of:

    Create Temporary Layer
    Save to File...
    Save to Geopackage...
    Save to PostGIS Table...
```
It looks a lot duplicated line.

**Comment by havatv** [comment]
> BTW, I was a little bit surprised that we need to repeat in every algorithm the description of the output:
> 
> ```
> Specify the output vector layer. One of:
> 
>     Create Temporary Layer
>     Save to File...
>     Save to Geopackage...
>     Save to PostGIS Table...
> ```
> 
> It looks a lot duplicated line.

We have several variants of that one, depending on the context (raster / vector / file / GDAL/OGR / ...).
I think it is important that the documentation of an algorithm is reasonably self-contained.  We could skip the "One of" part and provide a link instead, but that would be inconvenient for the user.
So providing the full explanation each time is a service to the user. To ease maintenance, we could try to use includes for this too. :-)

**Pull request merged**